src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -439,8 +439,11 @@ private void LookupExtensionMethodsInSingleBinder(ExtensionMethodScope scope, Lo
             Symbol symbolWithoutSuffix;
             bool resultWithoutSuffixIsViable = IsSingleViableAttributeType(result, out symbolWithoutSuffix);
 
-            // Generic types are not allowed.
-            Debug.Assert(arity == 0 || !result.IsMultiViable);
+            if (Compilation.LanguageVersion < LanguageVersion.CSharp8)
```

Comment:
The `if` and the one below should be bracketed inside an `#if DEBUG`.
Also, we should check for a feature flag (like `IDS_FeatureAutoPropertyInitializer`) rather than a language version. That new feature flag can be attached to CSharp8 for the purpose of this PR (mark with `PROTOTYPE(generic-attributes)`). But we'll want to adjust that before merging to a main branch.

Reply:
Where to mark the PROTOTYPE?

Reply:
@jcouv any other changes I need to do here?
didn't understand the PROTOTYPE thing